synapse/api/errors.py

```diff
@@ -55,6 +55,7 @@ class Codes(object):
     THREEPID_IN_USE = "M_THREEPID_IN_USE"
     THREEPID_NOT_FOUND = "M_THREEPID_NOT_FOUND"
     THREEPID_DENIED = "M_THREEPID_DENIED"
+    INVALID_THREEPID = "M_INVALID_THREEPID"
```

Comment:
It is unclear to me if this needs a spec change or not. @matrix-org/synapse-core any opinion?

Reply:
I did not find an spec for these errors. Only a documenttion https://github.com/matrix-org/matrix-doc/blob/master/specification/client_server_api.rst
I can also remove this error.

Reply:
You can see the list of error codes at https://matrix.org/docs/spec/client_server/r0.6.0#api-standards, this would need to get added as a separate one following the process at https://matrix.org/docs/spec/proposals